.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
```

Comment:
You may want to grab this section from the wezterm config; it installs a more deterministic version of rust and enables build caching:

https://github.com/wez/wezterm/blob/master/.github/workflows/gen_ubuntu16.yml#L22-L43

Reply:
TIL `actions/cache`. Thanks!

For `action-rs` I had some bad experience with it. It failed to download Rust (https://github.com/quark-zju/gcmodule/runs/487012737) and I stopped using it since then. So it might be more reliable to use the preinstalled rust.

Reply:
There's a lot of flakeyness in the GH Actions hosts :-/  I've had problems with random rust versions being installed; that normally doesn't matter except when you use something that was newly added to `stable` and the host is running the prior version.  Prior to using `actions-rs` I had some logic that downloaded and ran rustup directly.

If streampager isn't so sensitive to the rust version then just using the pre-installed rust should be fine!